tools/lexical-model.ts

```diff
@@ -0,0 +1,35 @@
+
+interface LexicalModel {
+  readonly format: 'trie-1.0'|'fst-foma-1.0'|'custom-1.0',
+  readonly wordBreaking?: {
+    allowedCharacters?: { initials?: string, medials?: string, finals?: string } | string,
+    defaultBreakCharacter?: string
+    sources?: Array<string>;
+  },
+  //... metadata ...
+}
+
+interface LexicalModelPrediction {
```

Comment:
Are these interfaces to coorespond with the interfaces in the main repo?

See: https://github.com/keymanapp/keyman/blob/master/common/predictive-text/message.d.ts#L145-L224

Reply:
Yes, this is really a stub for the real thing. I will update `LexicalModelPrediction` to correspond more cleanly with the `Transform` interface (but will not rely on the one in the main repo for now -- refactor can come once everything stabilises I think).